src/templates/_includes/nav_item.html

```diff
@@ -0,0 +1,13 @@
+<ul>
```

Comment:
Correct. I originally thought about combining <li> + <a> to a single element, but shadow DOM support is terrible so I bailed on that idea. Missed this file. Good call.